feat(n-card): adds `content-class` `footer-class` `header-class` ..... (#5262)
tusen-ai/naive-ui: src/card/demos/enUS/index.demo-entry.md

| --- | --- | --- | --- | --- |
| bordered | `boolean` | `true` | Whether to show the card border. |  |
| closable | `boolean` | `false` | Is it allowed to close. |  |

> | content-class | `string` | `undefined` | The class of the card content area. |  |

Reviewer: ```suggestion
| content-class | `string` | `undefined` | The class of the card content area. |  NEXT_VERSION |
```

| content-style | `Object \| string` | `undefined` | The style of the card content area. |  |
| embedded | `boolean` | `false` | Use a darker background color to show the embedding effect (only for bright themes) |  |

> | footer-class | `string` | `undefined` | The class of the bottom area of the card. |  |

Reviewer: ```suggestion
| footer-class | `string` | `undefined` | The class of the bottom area of the card. | NEXT_VERSION |
```

| footer-style | `Object \| string` | `undefined` | The style of the bottom area of the card. |  |

> | header-class | `string` | `undefined` | The class of the card head area. |  |

Reviewer: ```suggestion
| header-class | `string` | `undefined` | The class of the card head area. | NEXT_VERSION |
```

| header-style | `Object \| string` | `undefined` | The style of the card head area. |  |

> | header-extra-class | `string` | `undefined` | The class of the card head extra area. | 2.25.0 |

Reviewer: ```suggestion
| header-extra-class | `string` | `undefined` | The class of the card head extra area. | NEXT_VERSION |
```
